libs/island-ui/core/src/lib/Footer/Footer.tsx

```diff
@@ -161,16 +143,15 @@ export const Footer = ({
                         </Link>
                       </Text>
                     </Inline>
-                  </Box>
-                </Box>
+                  </Inline>
+                </div>
               </Box>
             </GridColumn>
             {showMiddleLinks ? (
```

Comment:
maybe we should try to stay consistent when there is only one render option? instead of returning null, might need to cast to boolean (!!)
```suggestion
            {showMiddleLinks && (
```